automated-record-card/index.html

```diff
@@ -0,0 +1,196 @@
+    <script src="https://unpkg.com/contentful-ui-extensions-sdk@3"></script>
+    <style>
+      .cf-form-field {
+        height: 220px;
+        margin-bottom: 0;
+      }
+
+      #thumbnail-preview {
+        float: right;
+        max-height: 200px;
+        max-width: 35%;
+      }
+
+      #thumbnail-preview div {
+        height: 200px;
+        width: 100%;
+        background: #ddd;
+      }
+
+      #thumbnail-preview img {
+        width: auto;
+        max-height: 200px;
+      }
+
+      input.cf-form-input {
+        width: 60%;
+      }
+    </style>
+  </head>
+  <body>
+
+    <div class="cf-form-field">
+      <div id="thumbnail-preview">
+        <div></div>
+      </div>
+      <input type="text" placeholder="Paste your recordID" class="cf-form-input"/>
+      <p>
+        <button id="find-record" class="cf-btn-secondary">Go!</button>
+        <button id="reset-record" class="cf-btn-secondary">Reset</button>
+      </p>
+      <div id="message" class="cf-form-hint"></div>
+    </div>
+
+    <script>
+      'use strict';
+
+      window.contentfulExtension.init(extension => {
+        extension.window.startAutoResizer();
+
+        const inputRecordID = document.querySelector('.cf-form-input');
+        const fieldRecordID = extension.field;
+        let valueRecordID = fieldRecordID.getValue();
+        validateInput();
+
+        if (extension.entry.fields.thumbnailUrl.getValue()) {
+          document.getElementById('thumbnail-preview').innerHTML = '<img src="' + extension.entry.fields.thumbnailUrl.getValue() + '&size=w200" alt="" style="width: auto; height: 100%;" />'
+        }
+
+        const detachValueChangeHandler = fieldRecordID.onValueChanged(valueChangeHandler);
+        inputRecordID.addEventListener('input', keyboardInputHandler);
+        inputRecordID.addEventListener('blur', validateInput);
+        window.addEventListener('onbeforeunload', unloadHandler);
+
+        function validateInput() {
+          if (valueRecordID !== undefined && valueRecordID !== inputRecordID.value) {
+            document.getElementById('find-record').click();
+          }
+        }
+
+        function valueChangeHandler (value) {
+          inputRecordID.value = value || '';
+        }
+
+        function keyboardInputHandler () {
+          const value = inputRecordID.value;
+          if (typeof value !== 'string' || value === '') {
+            fieldRecordID.removeValue();
+          } else {
+            fieldRecordID.setValue(value);
+          }
+        }
+
+        function unloadHandler () {
+          window.removeEventListener('onbeforeunload', unloadHandler);
+          inputRecordID.removeEventListener('input', keyboardInputHandler);
+          detachValueChangeHandler();
+        }
+
+        // find a value in one of the defined languages
+        function languageValue (value) {
+          const languageKeys = ['en', 'eng', 'def', 'und'];
```

Comment:
This `languageKeys` setting may be best moved into a [UI extension parameter option](https://www.contentful.com/developers/docs/references/content-management-api/#/reference/ui-extensions/configuration-parameters).

We could either do so now, or just add a TODO note here for future.